<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #E6E6FA;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>200</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:50px;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/logo.jpeg</normaloff>:/icon/logo.jpeg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>341</width>
     <height>200</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="password">
      <property name="styleSheet">
       <string notr="true">background:white;
font-size:12pt; 
color: rgb(0, 0, 0);

border:1px solid;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="email">
      <property name="styleSheet">
       <string notr="true">background:white;
font-size:12pt; 
color: rgb(0, 0, 0);

border:1px solid;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">background:transparent;
font-size:10pt;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Password : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="loginbutton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28, 77, 255);
  border: none;
  color: white;
  text-align: center;
  text-decoration: none;
  font-size: 16px;
  padding: 10px ;
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="createaccbutton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(130, 0, 98);
  border: none;
  color: white;
  text-align: center;
  text-decoration: none;
  font-size: 16px;
  padding: 10px;
</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background:transparent;
font-size:10pt; 
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>User Id : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(28, 77, 255);
  border: none;
  color: white;
  text-align: center;
  text-decoration: none;
  font-size: 16px;
  padding:6px ;
</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>email</tabstop>
  <tabstop>password</tabstop>
  <tabstop>loginbutton</tabstop>
  <tabstop>createaccbutton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
